hendelse/domain/src/main/kotlin/no/nav/su/se/bakover/hendelse/domain/SakOpprettetHendelse.kt

```diff
@@ -13,22 +13,27 @@ import java.util.UUID
  */
 // TODO jah: FLytt nærmere sak/sakRepo
 data class SakOpprettetHendelse private constructor(
-    override val hendelseId: UUID,
+    override val hendelseId: HendelseId,
     override val sakId: UUID,
     val fnr: Fnr,
     val opprettetAv: NavIdentBruker,
     override val hendelsestidspunkt: Tidspunkt,
     override val meta: HendelseMetadata,
 ) : Hendelse {
+    override val tidligereHendelseId: HendelseId? = null
     override val entitetId = sakId
 
     // Dette vil alltid være første versjon i en hendelsesserie for en sak.
     override val versjon = Hendelsesversjon(1L)
+    override fun compareTo(other: Hendelse): Int {
+        require(this.entitetId == other.entitetId && this.sakId == other.sakId)
```

Comment:
Er det ikke tilstrekkelig å sjekke at `entitetId` er lik for å godta at man kan sortere?  Ser at de er like her, så det burde vel ikke utgjøre noen forskjell, men tenkte kanskje vi bare kunne hatt en default implementasjon for alle hendelser som godtar sortering innenfor en gitt `entitiyId`